stable/rabbitmq/values.yaml

```diff
@@ -129,6 +129,11 @@ rabbitmq:
   extraConfiguration: |-
     #disk_free_limit.absolute = 50MB
     #management.load_definitions = /app/load_definition.json
+  
+  ## Configuration file content: advanced configuration
+  ## Use this as additional configuraton in classic config format (Erlang term configuration format) 
+  advancedConfiguration: |-
+  
```

Comment:
This line is not needed

Reply:
Should I remove it or let it in place as a comment similarly as priorityClassName or existingClaim?

Reply:
mmm I was talking about the line 136, I mean, there are 2 lines between `advancedConfiguration: |-` and the following useful line while with only one line is ok

Reply:
Should by fixed